README.md

```diff
@@ -14,117 +14,175 @@ Here’s why you should try Percona Everest:
+```
 
-2. Verify that you have access to the Kubernetes cluster that you want to use with Everest. By default, Everest uses the kubeconfig file available under `~/.kube/config`. 
+2. **Install the Percona Everest Helm Chart:**
 
-    To verify access to the Kubernetes cluster, run the following command:
-   
-    ```sh 
-    kubectl get nodes
-    ```
+```bash
+helm install everest-core percona/everest \
+--namespace everest-system \
+--create-namespace
+```
 
-## Install Percona Everest
+3. **Retrieve Admin Credentials:**
 
-To install and provision Percona Everest to Kubernetes:
+```bash
+kubectl get secret everest-accounts -n everest-system -o jsonpath='{.data.users\.yaml}' | base64 --decode | yq '.admin.passwordHash'
+```
 
+- Default username: **admin**
+- You can set a different default admin password by using the server.initialAdminPassword parameter during installation.
 
-1. Install the latest version of the Everest CLI by running the following commands:
+4. **Access the Percona Everest UI:**
 
-    **Linux and WSL**
-        
-    ```sh
-    curl -sSL -o everestctl-linux-amd64 https://github.com/percona/everest/releases/latest/download/everestctl-linux-amd64
-    sudo install -m 555 everestctl-linux-amd64 /usr/local/bin/everestctl
-    rm everestctl-linux-amd64
-    ```
+   By default, Everest is not exposed via an external IP. Use one of the following options:
 
-    **macOS (Apple Silicon)**
+- Port Forwarding:
 
-    ```sh
-    curl -sSL -o everestctl-darwin-arm64 https://github.com/percona/everest/releases/latest/download/everestctl-darwin-arm64
-    sudo install -m 555 everestctl-darwin-arm64 /usr/local/bin/everestctl
-    rm everestctl-darwin-arm64
-    ```
+```bash
+kubectl port-forward svc/everest 8080:8080 -n everest-system
+```
 
-    **macOS (Intel CPU)**
+Access the UI at http://127.0.0.1:8080.
 
-    ```sh
-    curl -sSL -o everestctl-darwin-amd64 https://github.com/percona/everest/releases/latest/download/everestctl-darwin-amd64
-    sudo install -m 555 everestctl-darwin-amd64 /usr/local/bin/everestctl
-    rm everestctl-darwin-amd64
-    ```
+For more information about our Helm charts, visit the official [Percona Everest Helm Charts repository](https://github.com/percona/percona-helm-charts/tree/main/charts/everest).
 
-2. Install Everest and provision the Kubernetes cluster using one of the following commands:
+## Install Percona Everest using CLI
 
-    ```sh
-    everestctl install
-    ```
+> **Note**:
+>
+> Google Container Registry (GCR) is scheduled to be deprecated and will officially shut down on March 18, 2025. All versions of Percona Everest prior to 1.4.0 depend on images hosted on GCR. After the shutdown date, downloading those images will fail.
+>
+> We strongly recommend upgrading to Percona Everest version 1.4.0 as soon as possible. For more details, refer to the [Container Registry Deprecation Documentation](https://cloud.google.com/artifact-registry/docs/transition/prepare-gcr-shutdown)
```

Comment:
Why was this note added in this section? This is not specific to the installation using the CLI we should move this elsewhere.